nautobot/ipam/models.py

```diff
@@ -99,6 +99,7 @@ class VRF(PrimaryModel):
     name = models.CharField(max_length=100, db_index=True)
     rd = models.CharField(
         max_length=VRF_RD_MAX_LENGTH,
+        # unique=True,
```

Comment:
What's the plan here?

Reply:
Should be removed. This is covered by the `unique_together` for this model